### PR TITLE
carlosmalt

### PR DESCRIPTION
**Pull request by carlosmalt**
Add Ivo Jimenez back into UCSC and add UCSC publications

### CODE REVIEW

File: _data/publications/jimenez-rescue-hpc18.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform
```
Comment (henryiii) [inline _data/publications/jimenez-rescue-hpc18.yml]:
```suggestion
title: "Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform"
```

I believe you can't put a `:` in a string unless it's quoted (or not next to a space, IIRC, so YAML can support urls)

File: _data/publications/david-precs19.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+title: Reproducible Computer Network Experiments: A Case Study Using Popper
```
Comment (henryiii) [inline _data/publications/david-precs19.yml]:
```suggestion
title: "Reproducible Computer Network Experiments: A Case Study Using Popper"
```

File: _data/publications/jimenez-rescue-hpc18.yml (hunk 1 above)
Comment (henryiii) [reply _data/publications/jimenez-rescue-hpc18.yml]: In general, long strings should probably be surrounded by quotes, with just short, simple strings plain.

File: _data/publications/liu-hpc-iodc19.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+title: MBWU: Benefit Quantification for Data Access Function Offloading
```
Comment (henryiii) [inline _data/publications/liu-hpc-iodc19.yml]:
```suggestion
title: "MBWU: Benefit Quantification for Data Access Function Offloading"
```

File: _data/publications/jimenez-rescue-hpc18.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+title: "Spotting Black Swans With Ease: The Case for a Practical Reproducibility Platform"
+link: http://doi.org/10.5281/zenodo.1488346
+date: Nov 11, 2018
+citation: Ivo Jimenez, Carlos Maltzahn, st Workshop on Reproducible, Customizable and Portable Workflows for HPC (ResCuE-HPC’18, co-located with SC’18), Dallas, TX, November 11, 2018.
```
Comment (henryiii) [inline _data/publications/jimenez-rescue-hpc18.yml]:
```suggestion
citation: "Ivo Jimenez, Carlos Maltzahn, st Workshop on Reproducible, Customizable and Portable Workflows for HPC (ResCuE-HPC’18, co-located with SC’18), Dallas, TX, November 11, 2018."
```

I don't think this is required to have quotes, but nice to have.

File: _data/publications/maricq-osdi18.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+title: Taming performance variability
+link: https://www.usenix.org/conference/osdi18/presentation/maricq
+date: Oct 8-10, 2018
+citation: Aleksander Maricq, Dmitry Duplyakin, Ivo Jimenez, Carlos Maltzahn, Ryan Stutsman, and Robert Ricci, 13th USENIX Symposium on Operating Systems Design and Implementation (OSDI’18), Carlsbad, CA, October 8-10, 2018.
```
Comment (henryiii) [inline _data/publications/maricq-osdi18.yml]:
```suggestion
citation: "Aleksander Maricq, Dmitry Duplyakin, Ivo Jimenez, Carlos Maltzahn, Ryan Stutsman, and Robert Ricci, 13th USENIX Symposium on Operating Systems Design and Implementation (OSDI’18), Carlsbad, CA, October 8-10, 2018."
```

**Comment by carlosmalt** [comment]
Not sure what's wrong here. When I run this locally, I get an error unrelated to my changes:

>Saving /Users/carlosmalt/git/iris-hep/iris-hep.github.io-source/_cache/publications/1763198.yml
>jekyll 3.8.6 | Error:  undefined method `key?' for false:FalseClass
>rake aborted!
>Command failed with status (1): [jekyll serve --incremental...]
>/Users/carlosmalt/git/iris-hep/iris-hep.github.io-source/Rakefile:77:in `jekyll'
>/Users/carlosmalt/git/iris-hep/iris-hep.github.io-source/Rakefile:16:in `block in <top (required)>'
>Tasks: TOP => serve
>(See full trace by running task with --trace)
>bash-5.0$ 

The `--trace` option doesn't seem to reveal anything useful.

**Comment by henryiii** [comment]
That's the same error in the logs. I've assigned this to myself; I need to improve the error when this happens so that it's clear what has gone wrong. I expect something it is looking for is missing.

**Comment by henryiii** [comment]
The error is because `uta-nsdi20.yml` is an empty file.

**Pull request merged**